server/raft.go

```diff
@@ -353,7 +356,7 @@ func (s *Server) startRaftNode(cfg *RaftConfig) (RaftNode, error) {
 		return nil, errNoPeerState
 	}
 
-	qpfx := fmt.Sprintf("RAFT [%s - %s] ", hash[:idLen], cfg.Name)
+	qpfx := fmt.Sprintf("[ACC:%s] RAFT '%s' ", accName, cfg.Name)
```

Comment:
Whatever we decide on format probably make a utility function so we only change once?

Reply:
Not all queues are associated with an account, such as the system queue, etc.. but we could have an util function for the queue names that do have an account for sure.